posts/covid-analysis/index.qmd

```diff
@@ -0,0 +1,184 @@
+---
+title: "How well prepared are we to analyse an influenza pandemic? A brief perspective on analysis conducted for UK government advisory groups during COVID-19"
+author:
+  - name: "Adam Kucharski"
+    orcid: "0000-0001-8814-9421"
+date: "2024-11-28"
+categories: [COVID, R, real-time analysis, pandemic preparedness, H5N1]
+format:
+  html:
+    toc: true
+---
+
+With [multiple reports](https://www.youtube.com/watch?v=i4frpQJMfn8) of influenza H5N1 cases that have no clear animal exposure, it is useful to consider what kinds of analysis would be required, and how easily this could be performed. As a starting point, this post reflections on some of the real-time work my colleagues and I contributed to inform the UK response to COVID-19.
```

Comment:
```suggestion
With [multiple reports](https://www.youtube.com/watch?v=i4frpQJMfn8) of influenza H5N1 cases that have no clear animal exposure, it is useful to consider what kinds of analysis would be required, and how easily this could be performed. As a starting point, this post reflects on some of the real-time work my colleagues and I contributed to inform the UK response to COVID-19.
```